Userland/Libraries/LibWeb/Layout/TreeBuilder.cpp

```diff
@@ -383,25 +383,18 @@ ErrorOr<void> TreeBuilder::create_layout_tree(DOM::Node& dom_node, TreeBuilder::
         auto& parent = *dom_node.layout_node();
 
         // If the box does not overflow in the vertical axis, then it is centered vertically.
-        auto table_computed_values = parent.computed_values().clone_inherited_values();
-        static_cast<CSS::MutableComputedValues&>(table_computed_values).set_display(CSS::Display::from_short(CSS::Display::Short::Table));
-        static_cast<CSS::MutableComputedValues&>(table_computed_values).set_height(CSS::Size::make_percentage(CSS::Percentage(100)));
-
-        auto cell_computed_values = parent.computed_values().clone_inherited_values();
-        static_cast<CSS::MutableComputedValues&>(cell_computed_values).set_display(CSS::Display { CSS::Display::Internal::TableCell });
-        static_cast<CSS::MutableComputedValues&>(cell_computed_values).set_vertical_align(CSS::VerticalAlign::Middle);
-
-        auto row_computed_values = parent.computed_values().clone_inherited_values();
-        static_cast<CSS::MutableComputedValues&>(row_computed_values).set_display(CSS::Display { CSS::Display::Internal::TableRow });
-
-        auto table_wrapper = parent.heap().template allocate_without_realm<BlockContainer>(parent.document(), nullptr, move(table_computed_values));
-        auto cell_wrapper = parent.heap().template allocate_without_realm<BlockContainer>(parent.document(), nullptr, move(cell_computed_values));
-        auto row_wrapper = parent.heap().template allocate_without_realm<Box>(parent.document(), nullptr, move(row_computed_values));
-
-        cell_wrapper->set_line_height(parent.line_height());
-        cell_wrapper->set_font(parent.font());
-        cell_wrapper->set_children_are_inline(parent.children_are_inline());
-        row_wrapper->set_children_are_inline(false);
+        auto flex_computed_values = parent.computed_values().clone_inherited_values();
+        static_cast<CSS::MutableComputedValues&>(flex_computed_values).set_display(CSS::Display { CSS::Display::Outside::Block, CSS::Display::Inside::Flex });
+        static_cast<CSS::MutableComputedValues&>(flex_computed_values).set_justify_content(CSS::JustifyContent::Center);
+        static_cast<CSS::MutableComputedValues&>(flex_computed_values).set_flex_direction(CSS::FlexDirection::Column);
+        static_cast<CSS::MutableComputedValues&>(flex_computed_values).set_height(CSS::Size::make_percentage(CSS::Percentage(100)));
```

Comment:
Seems like it would be more readable to do the `static_cast` when first assigning to `flex_computed_values` instead of every time it's used.